homeassistant/components/spc/__init__.py

```diff
@@ -16,61 +14,60 @@
 CONF_API_URL = "api_url"
 
 DOMAIN = "spc"
-DATA_API = "spc_api"
+DATA_KEY = "spc"
 
 SIGNAL_UPDATE_ALARM = "spc_update_alarm_{}"
 SIGNAL_UPDATE_SENSOR = "spc_update_sensor_{}"
-
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We shouldn't nuke this right away – rather, we should use `cv.deprecation` and give an acceptable deprecation period.

Reply:
If it's not too much work to add a migration step, sure why not. I don't know how many people are using this integration and a manual migration shouldn't take more than a minute or so... so should really be a minor inconvenience, breaking change or not. I tried to find some examples for config migration but wasn't able to find any unfortunately.

Reply:
SimpliSafe does this (for now):

https://github.com/home-assistant/core/blob/766a0032c9a6050bf345c844e2cabd291644ab9b/homeassistant/components/simplisafe/__init__.py#L190-L214

https://github.com/home-assistant/core/blob/766a0032c9a6050bf345c844e2cabd291644ab9b/homeassistant/components/simplisafe/config_flow.py#L96-L98

Reply:
Thanks, now fixed.